.github/workflows/reusable-release.yml

```diff
@@ -266,3 +266,52 @@ jobs:
       -
         name: Create deployment job
         run: dagger do -l error deploy -w "actions:deploy:\"${{ env.AWS_REGION }}\":prod:\"${{ env.SHA }}\":\"${{ env.SHA_TAG }}\":_" -p ${{ env.DAGGER_PLAN }}
+
+  publish_nightly:
+    runs-on: ubuntu-latest
+    if: ${{ inputs.release_type == 'nightly' }}
+    steps:
+      -
+        name: Setup Node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 20
+      -
+        name: Checkout nightly branch
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ inputs.branch || 'develop' }}
+          token: ${{ secrets.GH_TOKEN }}
+          # Ref: https://github.com/lerna/lerna/issues/2532
+          fetch-depth: 0
+          # We do not want to push these changes upstream
+          detach: true
+      -
+        name: Configure git
+        run: |
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+          git config user.name ${{ github.actor }}
+      -
+        name: Install Dagger
+        uses: dagger/dagger-for-github@v3
+        with:
+          install-only: true
+          version: "0.2.36"
+      -
+        name: Verify Docker image
```

Comment:
I think it's fine to not verify a nightly, but if it's easy to do, no harm

Reply:
Yes, super easy to do, and doesn't add too much time to the process either.